cirq-google/cirq_google/experimental/noise_models/validate_noise_model_from_calibration_test.py

```
@@ -0,0 +1,190 @@
+        targets: ['0_0'],
+        values: [{{
+            double_val: {p11_1}
+        }}]
+    }}, {{
+        name: 'single_qubit_p11_error',
+        targets: ['0_1'],
+        values: [{{
+            double_val: {p11_2}
+        }}]
+    }}, {{
+        name: 'single_qubit_p11_error',
+        targets: ['1_0'],
+        values: [{{
+            double_val: {p11_3}
+        }}]
+    }}, {{
+        name: 'single_qubit_readout_separation_error',
+        targets: ['0_0'],
+        values: [{{
+            double_val: .004
+        }}]
+    }}, {{
+        name: 'single_qubit_readout_separation_error',
+        targets: ['0_1'],
+        values: [{{
+            double_val: .005
+        }}]
+    }},{{
+        name: 'single_qubit_readout_separation_error',
+        targets: ['1_0'],
+        values: [{{
+            double_val: .006
+        }}]
+    }}, {{
+        name: 'single_qubit_idle_t1_micros',
+        targets: ['0_0'],
+        values: [{{
+            double_val: {t1_1}
+        }}]
+    }}, {{
+        name: 'single_qubit_idle_t1_micros',
+        targets: ['0_1'],
+        values: [{{
+            double_val: {t1_2}
+        }}]
+    }}, {{
+        name: 'single_qubit_idle_t1_micros',
+        targets: ['1_0'],
+        values: [{{
+            double_val: {t1_3}
+        }}]
+    }}]
+""",
+        v2.metrics_pb2.MetricsSnapshot(),
+    )
+
+    random_seed = 1  # to make sure values match between simulators
+
+    calibration = cirq_google.Calibration(_CALIBRATION_DATA)
+    output_df = validate_noise_model(calibration, seed=random_seed)
+
+    # Construct expected results
+    expected_p00 = np.mean([p00_1, p00_2, p00_3])
+    expected_p11 = np.mean([p11_1, p11_2, p11_3])
+    expected_xeb_fidelity = 1 - np.mean([xeb_1, xeb_2])
+    expected_t1 = np.mean([t1_1, t1_2, t1_3]) * 1000
+
+    num_qubits_xeb = 2
+    N_xeb = 2 ** num_qubits_xeb
+    expected_decay_constant = 1 - (1 - expected_xeb_fidelity) * N_xeb / (N_xeb - 1)
```

RE expected values: ideally this would be a hardcoded dataframe, rather than repeating the experiments performed in `validate_noise_model`. Otherwise, an error in e.g.  `cross_entropy_benchmarking` could go unnoticed, because both the test amd `validate_noise_model` would be affected in the same way.